FND/DatesAndTimes/FinancialDates.rdf

```diff
@@ -267,12 +267,37 @@ The nth day is an ordinal number, not a cardinal number. &apos;1&apos; means the
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>date time stamp</rdfs:label>
-		<skos:definition>a combination of a date, a time, and a time zone</skos:definition>
+		<skos:definition>combination of a date, a time, and a time zone</skos:definition>
 		<fibo-fnd-utl-av:synonym>time stamp</fibo-fnd-utl-av:synonym>
 		<fibo-fnd-utl-av:usageNote>&apos;hasDateTimeStampValue&apos; is omitted if the DateTimeStamp is not (yet) established.</fibo-fnd-utl-av:usageNote>
 		<fibo-fnd-utl-av:usageNote>Use DateTimeStamp when the time stamp includes a time zone field. Use DateTime for the combination of a Date and a time, without a time zone.</fibo-fnd-utl-av:usageNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-dt-fd;DatedCollectionConstituent">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-arr;CollectionConstituent"/>
+		<rdfs:label>dated collection constituent</rdfs:label>
+		<owl:equivalentClass>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-fd;hasObservedDateTime"/>
```

Comment:
so anything with an observed date is inferred to be a collection constituent? Even if not in a collection?
I don't think we even need this class (or its superclass) - what value do they provide?

Reply:
So - we created these explicitly as a part of the work we did with Dan, so that we could support time series, pricing, yield curves, etc.  I suppose we could decide those are out of scope for FIBO and eliminate this, but I'm not ready to do that without a serious conversation and without an alternative that does the same thing.  These were moved from Arrangements to address your concern about dependencies - in the issue you specifically suggested that, in fact.